chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/test/InstanceSupervisor.kt

```diff
@@ -46,37 +40,40 @@ object InstanceSupervisor {
      * Tear downs the instance and runs the provided actions.
      */
     fun setUpInstanceWith(vararg actions: InstanceAction) {
-        waitForIdleInstance().wait(Wait.For(60, TimeUnit.SECONDS))
+        waitForIdleInstance()
+                // .wait(Wait.For(60, TimeUnit.SECONDS))
         listOf(tearDown(), setInstanceBusy(), createSuperUser())
                 .plus(actions)
                 .forEach(InstanceAction::run)
     }
 }
 
-private fun waitForIdleInstance(): Future<Result<Boolean, Error>> = Futures.schedule {
-    generateSequence { isInstanceIdle() }
-            .map {
-                it.also {
-                    when {
-                        it is Result.Success && !it.value -> Thread.sleep(1_000)
-                        it is Result.Failure -> error(it.error)
-                    }
-                }
-            }
-            .filter { it is Result.Success && it.value }
-            .first()
-}
-
-private fun isInstanceIdle(): Result<Boolean, Error> = chatkitInstance.request(
-        options = RequestOptions("/users", "GET"),
-        tokenProvider = sudoTokenProvider,
-        responseParser = { it.parseAs<List<User>>() }
-).wait().map { users ->
-    users.firstOrNull { it.name == "lock" }?.takeUnless { it.wasCreatedLongerThan(5_000) } == null
-}
-
-private fun User.wasCreatedLongerThan(millisecondsAgo: Long) =
-        Date().time - created.time > millisecondsAgo
+private fun waitForIdleInstance() {}
+// : Future<Result<Boolean, Error>> = Futures.schedule {
```

Comment:
what's the motivation to comment this out? that should still be useful for setup of e2e tests in v2 and should compile now after the removal of the stores and up

Reply:
When I removed the User type this was unhappy - I've added back a `UserApiType` and set this section to use the returned `UserApiType` object.